### ISSUE TITLE
YouTube 17.26.2 + uYou 3.0 [Feature Requests]

### ISSUE THREAD

**Issue by arichorn** [MiRO92/uYou-for-YouTube#152]
**Changes for uYou 3.0** 
- YouTube 16.42.3 will now be on 17.26.2 and will be optimized for uYou’s new changes!
- Notifications Tab API, an api for users that want to add tweaks for uYou, this grants you the ability to give back the notifications at the bottom bar on the app if not liking it on the Navigation Bar at the top.
- Integrated Fake YouTube Version, if users have problems with uYou even on the newest version you should get a toggle to make it act like 16.42.3 if needed (PoomSmart already has this but just in case uYou has problems too)
- Better Download Speeds! the youtube-dl will be replaced with yt-dlp which is a faster part of the code from the YouTube app

**uYou changes for YouTube 17.26.2**
Disclaimer: this one is for the new pointless features youtube adds which needs to be toggled off if it’s not needed.
- Swipe-Up Gesture, this new gesture came out whenever 17.19.2 released and this is very uncomfortable towards some iPad users. You swipe up to go into full screen even though YouTube already had a gesture can take you into full screen by pinching your fingers on the screen and also swiping up could be accidentally toggled if you cleaned your screen so this one is important.

**bug fixes for uYou 3.0**
- Notification Crash, tapping notifications button on uYou 2.1 can crash
- YouTube Profile & Banner Customization Crash, changing your banner or profile picture on uYou 2.1 can crash aswell
- Upload Button cannot be repositioned on uYou 2.1 for some reason?

That’s all hope y’all like these requests MiRO92!

**Comment by OblackatO** [comment]
Do you think we could add this feature as well: https://github.com/MiRO92/uYou-for-YouTube/issues/150 ?

**Comment by arichorn** [comment]
Heck yeah! I have edited the message up top! MiRO92 will love these ideas!

**Comment by arichorn** [comment]
Just recently updated this due to YouTube’s newer release

**Comment by OblackatO** [comment]
@arichorn are you trying to implement all the features above?

**Comment by arichorn** [comment]
Nope, I’m not capable of any of this. If I learnt how to actually code and then I would actually be able to do this and also uYou isn’t Open Source so this explains why all of these are suggested anyways.

**Comment by arichorn** [comment]
I have changed a few things here! YouTube 17.30.1 and Settings Reposition suggestion! This will make uYou have easier access to settings instead of going into the uYou Tab to change stuff!

**Comment by arichorn** [comment]
I have recently updated this! Made it easier to read and added 2 needed features in this Request
1. iOS 16 Compatibility
2. iPhone Layout Toggle

**Comment by arichorn** [comment]
Guys I’m gonna make another topic but suggestion for uYou 2.2, there’s a crap ton of issues recently happening on 2.1 because of googles server side changes.
